### ISSUE TITLE
no valid OpenPGP data found

### ISSUE THREAD

**Issue by kasbohm** [nipy/heudiconv#79]
Hi, docker build is throwing an error for me at:
`apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80 0xA5D32F012649A5A9`

```
gpg: requesting key 2649A5A9 from hkp server pool.sks-keyservers.net
gpgkeys: key A5D32F012649A5A9 can't be retrieved
gpg: no valid OpenPGP data found.
```

Changing hkp://pool.sks-keyservers.net -> hkp://**p80**.pool.sks-keyservers.net
seems to resolve the problem.

**Comment by yarikoptic** [comment]
On Wed, 13 Sep 2017, Inge Amlien wrote:

>    Hi, docker build is throwing an error for me at:
>    apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80
>    0xA5D32F012649A5A9

For some reason all the servers shown to be less reliable than in the
past.  Swapping one server for another might help in a specific time
point.  Alternative way is to fetch the key from our main server 
http://neuro.debian.net/faq.html

wget -q -O- http://neuro.debian.net/_static/neuro.debian.net.asc | sudo apt-key add -

We never made it default since hoped that gpg servers would be proven to be
more reliable than our 15 years old server ;)

**Comment by satra** [comment]
we should update the dockerfile to use `neurodebian:stretch` as image and pushing the problem to neurodebian docker builds :)

**Comment by mgxd** [comment]
closed with #117
